blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -53,6 +53,11 @@ Breaking Changes
 Changes
 =======
 
+- Extended :ref:`table-functions-generate-series` to work with timestamp and
```

Comment:
I think we could merge this into the previous entry. Something like  `Added the INTERVAL datatype and extended generate_series to work with timestamps and this new INTERVAL type`